tests/functional/verification/issues/tree.py

```diff
@@ -0,0 +1,11 @@
+from nagini_contracts.adt import ADT
```

Comment:
Probably this one as well?

Reply:
This one is imported in 00120.py. Importing was one of the sources of name clashes. So, in order to make the name clear it is related to issue #120, I renamed it to import_for_00120.py. Just remembering that import statements don't support importing files whose name has leading zeros.